portfolio/src/main/webapp/map.js

```diff
@@ -1,6 +1,30 @@
 /** Creates a map and adds it to the page. */
 function createMap() {
-  const map = new google.maps.Map(
-      document.getElementById('map'),
-      {center: {lat: 37.422, lng: -122.084}, zoom: 16});
+    const map = new google.maps.Map(
+        document.getElementById('map'),
+        {center: {lat: 37.422, lng: -122.084}, zoom: 16});
+
+    const trexMarker = new google.maps.Marker({
+      position: {lat: 37.421903, lng: -122.084674},
+      map: map,
+      draggable: true,
+      animation: google.maps.Animation.DROP,
+      title: "Stan the T-Rex"
+    });
+
+    trexMarker.addListener("click", function() {
+        toggleBounce(trexMarker);
+    });
+}
+
+/**
+ * Generates the bouncing animation for marker.
```

Comment:
I think the comment does not reflect the method?
Doesn't look like we generate animation? Look like we toggle the animation of the marker on / off?

Reply:
Fixed